web/crux-ui/src/pages/notifications/[notificationId].tsx

```diff
@@ -61,6 +75,11 @@ const NotificationDetailsPage = (props: NotificationDetailsPageProps) => {
           },
         ]}
       >
+        {!editing && (
+          <DyoButton type="button" className="px-4 mr-4 whitespace-nowrap" onClick={() => throttle(onTestHook)}>
```

Comment:
this is working right now like: the user clicks the button and we delay the requests for `WEBHOOK_TEST_DELAY`. Was this the original intent?

Reply:
I moved this code from the edit card, so this is how it originally worked. But I also tested it and if I spam the button after the delay it only executes once, which I think is fine.

Reply:
allrighty